tele_giphy/game/templates/game/index.html

```diff
@@ -2,10 +2,31 @@
 {% load staticfiles %}
 
 {% block main_content %}
-<div class="container">
-  <div class="row">
-    <br>Welcome to da telephone giphy game, you are on index.
-    <img src="{% static 'img/giphy_logo_laser_black.gif' %}" alt="">
+<div class="row">
+  <div class="col-md-2"></div>
+  <div class="col-md-8">
+	  <p>TeleGiphy is based off the icebreaker game, “Telephone”, where an original message is passed along a chain of players through whispers, and is entertaining through the often very distorted message at the end.<br>TeleGiphy is a spin-off of the game that relies on Giphy’s magnificent random-ness when selecting gifs instead of whispers to distort messages.
+	  </p>
+	  <br>
+	  <!--<p>Powered by <a href="http://giphy.com/"><img src="{% static 'img/giphy_logo_laser_black.gif' %}" alt=""></a></p>-->
+	  <p><a href="/game/{{ token }}"><button type="button" class="btn btn-primary btn-lg btn-block">New game</button></a></p>
```

Comment:
Using the {% url %} tag would be a better idea https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#url